recipes-retroarch/retroarch-database/retroarch-database.bb

```diff
@@ -25,7 +25,8 @@ FILES:${PN}-cheats = "${RETROARCH_DATABASE_CHEATS_DIR}"
 FILES:${PN}-cursors = "${RETROARCH_DATABASE_CURSORS_DIR}"
 FILES:${PN}-titles = "${RETROARCH_DATABASE_TITLES_DIR}"
 
-FILES:${PN} += "${RETROARCH_DATABASE_DIR}"
+FILES:${PN} = ""
```

Comment:
Come to think about it, what is accrual default value if not specified

Reply:
Maybe this line can be dropped?

Reply:
Ok, I checked some usage of `ALLOW_EMPTY:${PN} = "1"` in poky, and found that it often goes with pair with `FILES:${PN} = ""`.